packages/toolkit/src/utils/parseContractFunctions.ts

```diff
@@ -192,3 +192,121 @@ export default function parseContractFunctions(
 
   return functions
 }
+
+/**
+ * Finds global storage variable declarations with initialized values, e.g 'int a = 1'.
```

Comment:
Added a small clarification :)

```suggestion
 * Finds global storage variable declarations with initialized values, excluding constants. E.g 'int a = 1'.
```

Reply:
:+1: